tests/e2e/modules/compliments_spec.js

```diff
@@ -1,81 +0,0 @@
-const helpers = require("../helpers/global-setup");
```

Comment:
why did you delete this file? I don't see that all tests from here moved to electron section so we lost these tests.